src/Explorer/Explorer.tsx

```diff
@@ -1754,9 +1754,9 @@ export default class Explorer {
         this.isNotebooksEnabledForAccount(true);
         return;
       }
-      const isAccountInAllowedLocation = !disallowedLocations.some(
-        (disallowedLocation) => disallowedLocation === databaseAccountLocation
-      );
+
+      // firstWriteLocation should not be disallowed
+      const isAccountInAllowedLocation = disallowedLocations.indexOf(firstWriteLocation) === -1;
       this.isNotebooksEnabledForAccount(isAccountInAllowedLocation);
```

Comment:
>isNotebooksEnabledForAccount [](http://example.com/codeflow?start=11&length=28)

can you add telemetry for how many ppl click this button and enable it?

Reply:
there is already telemetry available for "CreateNotebookWorkspace" action which is triggered while enabling notebooks.